### PR TITLE
Connect firebase to the frontend

### PR DESCRIPTION
**Pull request by abbyjng**
Added Firebase files and modules, followed instructions to integrate React Native Firebase into the front end

### CODE REVIEW

**Comment by abbyjng** [comment]
> * Can you add some unit tests to confirm that you can actually access firebase? Even if things are working in theory, we can't really confirm without tests so it's risky to check in these files blindly.

Both Android and iOS were checked by connecting them with the Firebase server directly upon running and were confirmed to be connected — to be honest, I’m not sure how to make unit tests for this. Do you have any guidance on how to confirm this beyond the actual website check?

**Comment by shlee2017** [comment]
> 1. Are the xcode files necessary? Some of the negation stuff in the gitignore may not be working. @shlee2017 may know more

Any build and pod files are not necessary. There might be more files we can ignore, but not entirely sure.

**Comment by abbyjng** [comment]
> Any build and pod files are not necessary. There might be more files we can ignore, but not entirely sure.

Does that mean that the xcode files should be removed? They are in the main repository right now, but I can remove them with this PR if they shouldn't be.

**Comment by shlee2017** [comment]
> > Any build and pod files are not necessary. There might be more files we can ignore, but not entirely sure.
> 
> Does that mean that the xcode files should be removed? They are in the main repository right now, but I can remove them with this PR if they shouldn't be.

I think so since the react start or run iOS command should create it for you.

File: front_end/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "react": "16.11.0",
     "react-native": "0.62.2",
     "react-native-read-more-text": "^1.1.2"
```
Comment (xumaple) [inline front_end/package.json:15]: missing comma

**Comment by shapedapp** [comment]
Merging without approvals because approvers became stale with rebase

**Pull request merged**